plugins/reporters/web-app-template/src/App.jsx

```diff
@@ -17,252 +17,143 @@
  * License-Filename: LICENSE
  */
 
-import { Component } from 'react';
-
-import {
-    ControlOutlined,
-    PartitionOutlined,
-    PieChartOutlined,
-    TableOutlined
-} from '@ant-design/icons';
 import {
-    Alert,
-    Col,
-    Progress,
-    Row,
-    Tabs
-} from 'antd';
-import PropTypes from 'prop-types';
+    useEffect,
```

Comment:
Commit message:
- "with React*'s* own"
- "State handling per instance of a component possible" -> "Allow for handling per instance of a component"
- "It easier for new contributors to contribute" -> "Make it easier for new contributors to contribute"
- "Tree" -> "tree"
- "be restore" -> "be restored"